app/js/tabs.js

```diff
@@ -317,33 +325,68 @@ const addTab = options => {
     } else {
         tabs.appendChild(tab)
     }
-    const webview = document.createElement("webview")
-    webview.setAttribute("link-id", linkId)
     tab.setAttribute("link-id", linkId)
-    if (SETTINGS.get("spell")) {
-        webview.setAttribute("webpreferences", "spellcheck=yes")
-    }
     const color = SETTINGS.get("containercolors").split(",").find(
         c => sessionName.match(c.split("~")[0]))
     if (color) {
         tab.style.color = color.split("~")[1]
     }
-    SESSIONS.create(sessionName)
-    webview.setAttribute("partition", `persist:${sessionName}`)
-    webview.setAttribute("container", sessionName)
-    pages.appendChild(webview)
+
+    const page = document.createElement("div")
+    page.classList.add("webview")
+    page.setAttribute("link-id", linkId)
+    if (options.url) {
+        page.src = UTIL.stringToUrl(options.url)
+    }
+    page.callback = options.callback
+    page.setAttribute("container", sessionName)
+    if (isDevtoolsTab) {
+        page.setAttribute("devtools-for-id", currentPageId)
+    }
+    page.setAttribute("muted", options.muted)
+    pages.appendChild(page)
+    if (options.lazy) {
+        const url = UTIL.stringToUrl(options.url)
+        tab.setAttribute("suspended", "suspended")
+        title.textContent = HISTORY.titleForPage(url) || url
+        favicon.src = FAVICONS.forSite(url) || favicon.src
+    } else {
+        unsuspendPage(page)
+    }
     if (options.switchTo) {
         switchToTab(listTabs().indexOf(tab))
     }
+}
+
+const unsuspendPage = page => {
+    if (page.tagName === "WEBVIEW") {
+        return
+    }
+    tabOrPageMatching(page).removeAttribute("suspended")
+    const webview = document.createElement("webview")
+    ;["link-id", "container", "class", "id", "style"].forEach(
+        attr => webview.setAttribute(attr, page.getAttribute(attr)))
```

Comment:
I think we can just add muted to this list to read it, and expand this to filter `null` attributes before setting them, like this:
```
;["link-id", "container", "class", "id", "style", "muted"].forEach(attr => {
    if (page.getAttribute(attr)) {
        webview.setAttribute(attr, page.getAttribute(attr))
    }
})
```